sdl2promela/sdl/model.py

```diff
@@ -1451,6 +1451,54 @@ def convert(source: ogAST.Decision) -> Action:
     return None
 
 
+def _parseObserverInput(desc: str) -> Tuple[ObserverAttachmentInfo, str]:
```

Comment:
The name of the function might be misleading. Can we indicate that we are parsing a continuous signal?

Reply:
This could also differentiate it from __parse_observer_input

Reply:
Done